rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_reader :book, :student
+  attr_accessor :date
+
+  def initialize(date, book, student)
```

Comment:
- [ ] please kindly note A rental belongs to a book and also belongs to a person instead of student. so please @person should be present as one of the instance variables .....  `:date, :book, :person`
- [ ] please book and person should be set in the rental constructor so as to break the many-to-many relationship 
`def initialize(date, book, person)`